speech/samples/v1p1beta1/samplegen_read_and_write_files.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "samplegen_read_and_write_files")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-speech
+
+# sample-metadata
+#   title: Read binary file into bytes field & write string in response to file
+#   description: Read binary file into bytes field & write string in response to file
+#   usage: python3 samples/v1p1beta1/samplegen_read_and_write_files.py
+
+# [START samplegen_read_and_write_files]
+from google.cloud import speech_v1p1beta1
+from google.cloud.speech_v1p1beta1 import enums
```

Comment:
I prefer importing modules over members.

```
from google.cloud import speech_v1p1beta1
```

This helps to avoid alias collisions should people be importing multiple client libraries.

Reply:
Ack. Thanks for pointing this out!

If I remember correctly, some LOC can be very long with `<module>.<enums>.` for each enum. If the generator were smart enough, it could decide when [not] to import enums.

Reply:
@gguuss @leahecole et al

Does this fall into `nit` or `blocker` category?

And are we sure that this change should be made?

I've seen a preference _for_ importing `enums` – it's a style I've seen in a bunch of code samples. In [`python-docs-samples`](https://github.com/GoogleCloudPlatform/python-docs-samples/search?q=%22import+enums%22&unscoped_q=%22import+enums%22), you can find the `import enums` style used for all of these 10 APIs:
 - Asset
 - BigTable
 - KMS
 - Natural Language
 - Speech-to-Text
 - AutoML Tables
 - Text-to-Speech
 - Translation
 - Video Intelligence
 - Vision

[Search for "import enums"](https://github.com/GoogleCloudPlatform/python-docs-samples/search?q=%22import+enums%22&unscoped_q=%22import+enums%22)